src/_imaging.c

```diff
@@ -433,6 +433,36 @@ float16tofloat32(const FLOAT16 in) {
     return out[0];
 }
 
+static inline PyObject *
+getpixel_mb(Imaging im, ImagingAccess access, int x, int y) {
+    UINT8 pixel[sizeof(INT32) * 6];
```

Comment:
This used to be `UINT8 pixel[im->pixelsize]` but a strict compiler is unhappy.
Currently the largest entry in `OPEN_INFO` has 6 bands, so that it's defined as 6.
I may have to put a pre-allocated `uint8[pixelsize]` buffer in `Imaging` struct.